README.md

```diff
@@ -36,7 +36,11 @@ The Infusion library is the cornerstone of a number of projects. You can see som
 
 ## How Do I Create an Infusion Package? ##
 
-Strictly speaking, Infusion can be used directly from source (i.e. by including each individual required file). However, for simplicity and performance reasons, you may wish to create a concatenated, minified file. The Grunt build options described below will also allow you to remove any unneeded features or libraries that you may already have in your project.
+For simplicity and performance reasons, you may wish to create a concatenated, minified file. However, such a file is often difficult to read. To address this, source maps for the minified file are automatically generated to make debugging easier.
+
+### Source Maps ###
+
+Source maps are supported in all of the major browser: [Chrome](https://developer.chrome.com/devtools/docs/javascript-debugging#source-maps), [Firefox](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map), [IE 11](https://msdn.microsoft.com/library/dn255007#source_maps), and Safari. To make use of them, enabled source maps in your support debugging environment, and ensure that the source maps are hosted from the same relative path that they were generated at.
```

Comment:
typos browser - > browsers
enabled -> enable

Could you flesh out this description a little further? "hosted from the same relative path that they were generated at" isn't completely clear to me. Do you think you could write out a set of concrete, step-by-step instructions on how to use a source map in at least one situation on one browser? Starting from the initial build steps with the commands and options to be used, to the include structure of the file and putting the files in the right places, and finishing with steps in a particular browser and what one should expect to see